test-framework/provisioning/roles/barnacle/tasks/main.yml

```diff
@@ -60,16 +70,14 @@
     register: stacki_fab_rpm
 
   - name: Install foundation-python
-    command: "rpm -ivh {{ item.path }}"
+    command: "rpm -ivh {{ foundation_python_rpm.files[0].path }}"
```

Comment:
I like this better. Thanks for cleaning it up.

Reply:
Your way was good too, but this way the output is:
```
TASK [barnacle : Find the foundation-python RPM in the stacki ISO] *************
ok: [frontend]

TASK [barnacle : Find the stacki-fab RPM in the stacki ISO] ********************
ok: [frontend]
```

Instead of:
```
TASK [barnacle : Install foundation-python] ************************************
changed: [frontend] => (item={'uid': 0, 'woth': False, 'mtime': 1582802009.0, 'inode': 2315, 'isgid': False, 'size': 26703783, 'roth': True, 'isuid': False, 'isreg': True, 'pw_name': 'root', 'gid': 0, 'ischr': False, 'wusr': False, 'xoth': False, 'rusr': True, 'nlink': 3, 'issock': False, 'rgrp': True, 'gr_name': 'root', 'path': '/mnt/stacki/stacki/5.4.1_20200227_b927608/sles12/sles/x86_64/RPMS/foundation-python-3.8.1-sles12.x86_64.rpm', 'xusr': False, 'atime': 1582802928.0, 'isdir': False, 'ctime': 1582802928.0, 'isblk': False, 'xgrp': False, 'dev': 1792, 'wgrp': False, 'isfifo': False, 'mode': '0444', 'islnk': False})

TASK [barnacle : Install stacki-fab] *******************************************
changed: [frontend] => (item={'uid': 0, 'woth': False, 'mtime': 1582803106.0, 'inode': 2726, 'isgid': False, 'size': 7209, 'roth': True, 'isuid': False, 'isreg': True, 'pw_name': 'root', 'gid': 0, 'ischr': False, 'wusr': False, 'xoth': False, 'rusr': True, 'nlink': 1, 'issock': False, 'rgrp': True, 'gr_name': 'root', 'path': '/mnt/stacki/stacki/5.4.1_20200227_b927608/sles12/sles/x86_64/RPMS/stacki-fab-5.4.1_20200227_b927608-sles12.x86_64.rpm', 'xusr': False, 'atime': 1582803107.0, 'isdir': False, 'ctime': 1582803106.0, 'isblk': False, 'xgrp': False, 'dev': 1792, 'wgrp': False, 'isfifo': False, 'mode': '0444', 'islnk': False})
```